test/e2e/testsuites/dynamically_provisioned_job_attach_batch_tester.go

```diff
@@ -97,9 +98,21 @@ func (t *DynamicallyProvisionedAttachBatchTest) Run(ctx context.Context, client
 			framework.ExpectNoError(err)
 		}
 
+		// check if all jobs have completed, if failed, then wait for 1min, and check again with maximum timeout
 		ginkgo.By("checking if there are any jobs left")
 		tjobs, err := client.BatchV1().Jobs(namespace.Name).List(context.TODO(), metav1.ListOptions{})
-		framework.ExpectNoError(err)
+		for i := 0; i < 15; i++ {
```

Comment:
If we want to extend the timeout, can we adjust the timeout here https://github.com/hccheng72/azuredisk-csi-driver/blob/5175bf0edfa67f7ee2d10b87dcb990a393ec25e8/test/e2e/testsuites/testsuites.go#L896, so it can keep detecting if the MaximumDataDisksExceeded occurs before timeout?

Reply:
it's already waiting another 7min, and the issue is not fixed.